rpm/containers-common.spec

```diff
@@ -0,0 +1,237 @@
+# Below definitions are used to deliver config files from a particular branch
+# of c/image, c/storage  and c/shortnames vendored in all of Buildah, Podman and Skopeo.
+# These vendored components must have the same version. If it is not the case,
+# pick the oldest version on c/image, c/storage and c/shortnames vendored in
+# Buildah/Podman/Skopeo.
+
+# Packit will automatically update the image and storage versions on Fedora and
+# CentOS Stream dist-git PRs.
+%global image_branch main
+%global storage_branch main
+%global shortnames_branch main
+
+%global github_containers https://raw.githubusercontent.com/containers
+
+%if %{defined copr_username}
+%define copr_build 1
+%endif
+
+Name: containers-common
+%if %{defined copr_build}
+Epoch: 102
+%else
+Epoch: 5
+%endif
+# DO NOT TOUCH the Version string!
+# The TRUE source of this specfile is:
+# https://github.com/containers/common/blob/main/rpm/containers-common.spec
+# If that's what you're reading, Version must be 0, and will be updated by Packit for
+# copr and koji builds.
+# If you're reading this on dist-git, the version is automatically filled in by Packit.
+Version: 0
+Release: %autorelease
+License: Apache-2.0
+BuildArch: noarch
+# for BuildRequires: go-md2man
+ExclusiveArch: %{golang_arches} noarch
+Summary: Common configuration and documentation for containers
+BuildRequires: git-core
+BuildRequires: go-md2man
+Provides: skopeo-containers = %{epoch}:%{version}-%{release}
+Requires: (container-selinux >= 2:2.162.1 if selinux-policy)
+Suggests: fuse-overlayfs
+Source0: %{git0}/archive/v%{version_no_tilde}.tar.gz
```

Comment:
Have you verified that `version_no_tilde` works as intended? I mean, it _has_ to, otherwise I don't think any of this would work... but it seems to be defined in `/usr/lib/rpm/macros.d/macros.rust-srpm` and I don't know enough about RPM macro processing to know if that file is in the standard include path, or if it exists in RHEL/CentOS, or lots of questions. (Same macro is used in podman, so yeah this really seems like it works, but I want to make triple-sure)

Reply:
it's already there in podman's spec file and has been working well for podman RCs and stable releases. I included `version_no_tilde` to account for any possible RCs we may wanna do in the future.

Reply:
Besides, centos 10 packit workflow still needs a manual `centpkg build` after dist-git PR merge, so this is a pretty safe change.